Update glide files and vendor directory.

craigyam opened the pull request:
Need to add a dependency to the glide.yaml and update.
Adding golang.org/x/net/context (for future use) and updated the glide.yaml, glide.lock, and vendor directory.

craigyam committed:
Update glide files and vendor directory.

Need to add a dependency to the glide.yaml and update.

rshriram commented:
@craigyam given that the PR is too big for display in GH, can you please dump the diff for glide.yaml and glide.lock in the comments please?

craigyam commented:
glide.lock:

```
-hash: 5028fee619e8faf8194772aa8d0fd803a739c680736b02debd7f2fd8cebd073c
 -updated: 2016-09-07T14:57:14.651910967+03:00
 +hash: 0a333ffd3a7b7056d1b1a98e1df73fd28736f2f4cb2f7876af5af324d26adeaf
 +updated: 2016-10-12T22:20:41.443793286+03:00
  imports:
  - name: github.com/ant0ine/go-json-rest
    version: 37ad7dc626602a8355801da6acd85155655520f8
 @@ -26,35 +26,8 @@ imports:
    - i18n/translation
  - name: github.com/onsi/ginkgo
    version: 462326b1628e124b23f42e87a8f2750e3c4e2d24
 -  subpackages:
 -  - config
 -  - internal/codelocation
 -  - internal/failer
 -  - internal/remote
 -  - internal/suite
 -  - internal/testingtproxy
 -  - internal/writer
 -  - reporters
 -  - reporters/stenographer
 -  - types
 -  - internal/containernode
 -  - internal/leafnodes
 -  - internal/spec
 -  - internal/specrunner
  - name: github.com/onsi/gomega
    version: 0fe204460da2c8fa1babcaac196e694de8f1aaa1
 -  subpackages:
 -  - internal/assertion
 -  - internal/asyncassertion
 -  - internal/testingtsupport
 -  - matchers
 -  - types
 -  - internal/oraclematcher
 -  - format
 -  - matchers/support/goraph/bipartitegraph
 -  - matchers/support/goraph/edge
 -  - matchers/support/goraph/node
 -  - matchers/support/goraph/util
  - name: github.com/pborman/uuid
    version: dee7705ef7b324f27ceb85a121c61f2c2e8ce988
  - name: github.com/rcrowley/go-metrics
 @@ -67,11 +40,6 @@ imports:
    - formatters/logstash
  - name: github.com/stretchr/testify
    version: 9f9027faeb0dad515336ed2f28317f9f8f527ab4
 -  subpackages:
 -  - assert
 -  - mock
 -  - require
 -  - suite
  - name: github.com/urfave/cli
    version: 168c95418e66e019fe17b8f4f5c45aa62ff80e23
  - name: github.com/xeipuuv/gojsonpointer
 @@ -80,16 +48,10 @@ imports:
    version: e02fc20de94c78484cd5ffb007f8af96be030a45
  - name: github.com/xeipuuv/gojsonschema
    version: 0a98b2bd93655a563d1f23c4407e8f01791b6b31
 -- name: gopkg.in/yaml.v2
 -  version: e4d366fc3c7938e2958e662b4258c7a89e1f0e3e
 -testImports:
 -- name: github.com/davecgh/go-spew
 -  version: 2df174808ee097f90d259e432cc04442cf60be21
 -  subpackages:
 -  - spew
 -- name: github.com/pmezard/go-difflib
 -  version: d8ed2627bdf02c080bf22230dbb337003b7aba2d
 +- name: golang.org/x/net
 +  version: 6dba816f1056709e29a1c442883cab1336d3c083
    subpackages:
 -  - difflib
 -- name: github.com/stretchr/objx
 -  version: cbeaeb16a013161a98496fad62933b1d21786672
 +  - context
 +- name: gopkg.in/yaml.v2
 +  version: a5b47d31c556af34a302ce5d659e6fea44d90de0
 +devImports: []
```

craigyam commented:
glide.yaml:

```
subpackages:
    - statsd

 +- package: golang.org/x/net/context
 +  version: 6dba816f1056709e29a1c442883cab1336d3c083
 +
  - package: github.com/pborman/uuid
    version: dee7705ef7b324f27ceb85a121c61f2c2e8ce988

```
